### ISSUE TITLE
The Datadog class is missing from 1.7.0 in the Python cdk-v2 library

### ISSUE THREAD

**Issue by kcp-chewie** [DataDog/datadog-cdk-constructs#180]
## Expected Behavior
`from datadog_cdk_constructs_v2 import Datadog` works.

## Actual Behavior
`from datadog_cdk_constructs_v2 import Datadog` throws an import error.
  
## Stacktrace
  
  ```
Traceback (most recent call last):
  File "/home/runner/work/xxxx/xxxx/app.py", line 7, in <module>
    from infra import common
  File "/home/runner/work/xxxx/xxxx/infra/common.py", line 11, in <module>
    from xxx.cdk.datadog import Monitoring as DatadogMonitoring
  File "/opt/hostedtoolcache/Python/3.9.16/x64/lib/python3.9/site-packages/xxx/cdk/datadog.py", line 14, in <module>
    from datadog_cdk_constructs_v2 import Datadog
ImportError: cannot import name 'Datadog' from 'datadog_cdk_constructs_v2' (/opt/hostedtoolcache/Python/3.9.16/x64/lib/python3.9/site-packages/datadog_cdk_constructs_v2/__init__.py)  ```

**Comment by thedavl** [comment]
Hi @kcp-chewie, thanks for bringing this up. We're working on a fix now, but in the meantime please use version `1.61.0`

**Comment by thedavl** [comment]
This has been fixed with version `1.71.0`

Please reach out if you have any other issues!
